browser/css/notebookbar.css

```diff
@@ -336,6 +336,16 @@ html[data-theme='dark'] .savemodified.unotoolbutton .unobutton img {
 	grid-column: 2;
 }
 
+.ui-checkbox {
+	display: flex;
+	flex-direction: row;
+}
+
+.ui-checkbox-label {
```

Comment:
`label.notebookbar.ui-checkbox-label` will do the trick, so you don't need the `:not` above